config/mgc-install-guide/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+resources:
+  - ../cert-manager
```

Comment:
is there a way to optionally include this? Some clusters may already have cert manager installed

Reply:
I'll try moving installation of cert-manager to the "Creating a Cert Issuer" step and mark it as optional.